### PR TITLE
Fix root directory problem

### PR DESCRIPTION
**Pull request by Yu-Che-Gao**
If root directory is not "/", this plugin has some problems. This commit fix this.

### CODE REVIEW

**Comment by ele828** [comment]
Thanks for your contribution!
What about javascript importing statements below?

File: index.js [hunk 1]
```diff
@@ -112,11 +112,11 @@ function copyAssets() {
...
-    `<link rel="stylesheet" href="/css/${prismThemeFileName}" type="text/css">`
+    `<link rel="stylesheet" href="` + hexo.config.root + `css/${prismThemeFileName}" type="text/css">`
   ];
 
   if(line_number) {
-    css.push(`<link rel="stylesheet" href="/css/prism-line-numbers.css" type="text/css">`);
+    css.push(`<link rel="stylesheet" href="` + hexo.config.root + `css/prism-line-numbers.css" type="text/css">`);
   }
   if(mode === 'realtime') {
     js.push('<script src="/js/prism.js"></script>');
```
Comment (ele828) [inline index.js]: Do we also need to change it here?

Comment (ele828) [inline index.js]: Also on line 122

Comment (Yu-Che-Gao) [inline index.js]: Yes, you need to change it on line 122 and 124 to fix the problem of root directory.

Comment (ele828) [inline index.js]: I'd appreciate it if you could help fix it together in this PR, so that I can merge it entirely in one time :)

Comment (Yu-Che-Gao) [inline index.js]: Ok, Just wait a moment.

Comment (Yu-Che-Gao) [inline index.js]: I have changed code. Maybe you can check it and test it then merge my code.

**Comment by ele828** [comment]
Thank you!

**Pull request merged**